HISTORY.md

```diff
@@ -1,3 +1,63 @@
+# Assistify 0.9.9
```

Comment:
*NO PR*

We add our Assistify.Chat-history always on top of the last entry. This keeps the diff cleaner